apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt

```diff
@@ -40,44 +40,54 @@ import java.net.URI
 @Composable
 fun NewChatSheet(rh: RemoteHostInfo?, close: () -> Unit) {
   val oneHandUI = remember { chatModel.controller.appPrefs.oneHandUI }
+  val showToolbarInOneHandUI = remember { mutableStateOf(oneHandUI.state.value) }
+  val keyboardState by getKeyboardState()
 
-  Column(
-    modifier = Modifier.fillMaxSize()
-  ) {
-    if (!oneHandUI.state.value) {
-      Box(contentAlignment = Alignment.Center) {
-        val bottomPadding = DEFAULT_PADDING
-        AppBarTitle(
-          stringResource(MR.strings.new_chat),
-          hostDevice(rh?.remoteHostId),
-          bottomPadding = bottomPadding
+  LaunchedEffect(keyboardState, oneHandUI) {
```

Comment:
You are doing something strange here. On line 43 you remember SharedPreferences which will never be changed. Intead, you need to remember state like so:
```kotlin
  val oneHandUI = remember { chatModel.controller.appPrefs.oneHandUI.state }

```
And write in LaunchedEffect like so:

```kotlin
  LaunchedEffect(keyboardState, oneHandUI.value) {

```